src/core/calculate.js

```diff
@@ -62,7 +62,7 @@ const calculateForAST = (selectorAST) => {
                     case 'nth-last-child':
                         b += 1;
 
-                        if (child.children && child.children.first.selector) {
+                        if (child.children && child.children.first && child.children.first.selector) {
```

Comment:
I think we can simplify this (and the other ones below) to:

```suggestion
                        if (child.children?.first?.selector) {
```

Reply:
Yeah, I agree. Just tried to keep a consistent style, but this is better 🙈